src/Pages/ProductList.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { StyledFlexCenteredContainer, BlockedStyledContainer } from '../utils/StyledComponents/Styled';
+import Sidebar from '../Components/Sidebar';
+import ProductGrid from '../Components/ProductsGrid';
+import Pagination from '../Components/Pagination';
+
+export default function ProductList() {
+
+  const [categories, setCategories] = useState([]);
```

Comment:
Isnt it possible to handle both web and mobile in same variable? this way you can make your code cleaner.